app/bundles/LeadBundle/Entity/Lead.php

```diff
@@ -317,6 +324,16 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
             ->cascadeAll()
             ->fetchExtraLazy()
             ->build();
+
+        $builder->createManyToMany('companies', 'Mautic\LeadBundle\Entity\Company')
```

Comment:
Should probably use a `->fetchExtraLazy()` here as well unless we think there's a chance that Company will be used for every Lead object queried. Otherwise, Doctrine is going to query for Company and CompanyLead automatically when hydrating Lead objects.